File: 프로그래머스/0/181874. A 강조하기/A 강조하기.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>
#include <stdlib.h>

char* solution(const char* myString) {
    char* answer = (char*)malloc(sizeof(char)*(strlen(myString)+1));
    memcpy(answer, myString, strlen(myString)+1);
    for(int i=0;i<strlen(answer);i++){
        if(answer[i]=='a'||answer[i]=='A'){answer[i]='A';}
        else{answer[i]=tolower(answer[i]);}
    }
    return answer;
}

File: 프로그래머스/0/181874. A 강조하기/A 강조하기.py
def solution(myString):
    result = ''
    for char in myString:
        if char == 'a':
            result += 'A'
        elif char == 'A':
            result += 'A'
        elif char.isupper():
            result += char.lower()
        else:
            result += char
    return result
